Modix/ClientApp/src/components/PieChart.vue

```diff
@@ -10,6 +10,7 @@ import { Chart } from 'chart.js';
 
```

Comment:
During testing, I was getting a bug where the pie chart wouldn't update if you looked up a different user. I think we can fix this by putting `@Watch('stats')` above the `updateChart` function. That way, it should automagically update the chart whenever the stats change

Reply:
yes, fixed

`mounted()` will be always called before `@Watch()` right?

Reply:
I think so. I'm not really a Vue expert, but if I remember correctly, `mounted()` happens once right after the component is initially rendered, and then `Watch` happens every time a property is updated